src/avfoundation.cs

```diff
@@ -8058,6 +8176,10 @@ interface AVCaptureSession {
 		[iOS (10, 0)]
 		[Export ("automaticallyConfiguresCaptureDeviceForWideColor")]
 		bool AutomaticallyConfiguresCaptureDeviceForWideColor { get; set; }
+
+		[iOS (11, 1)]
```

Comment:
the `#if` are not clear in the diff but this cannot be added in Mac, TV or Watch since the headers were not updated

Reply:
This is within a #if !MONOMAC in line 8156 to line 8184 that includes the only iOS methods. The class has a NoWatch NoTV already, therefore AFAIK, it is not added in Mac, TV or Watch.

Reply:
I agree that because the class has `[NoWatch]` and `[NoTV]` we shouldn't add those but I would still add the `[NoMac]` even though there's a `#if !MONOMAC`. Remember we're trying to move away from them, it adds clarity.

Reply:
even a `[NoMac]` should be added on the type, so we would not see it in the diff :|
hopefully xtro will make this easier to review (really soon now)

Reply:
Adding NoMac and moving out side the #if !MONOMAC.